Product/ProductTests/ProductTests.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  ProductTests.swift
+//  ProductTests
+//
+//  Created by Dharmesh Patel on 05/04/20.
```

Comment:
Please change the name

Reply:
This name is okay what is the issue in the name?